app/js/components/accordion/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import '../../../css/accordion.css';
```

Comment:
Could this implementation be added as part of the `openmrs-owa-orderentry.css` which is bundled by webpack so we don't have to be making individual imports?

Reply:
Great suggestion but my concern with that approach is about the conflicts that might arise if/when there are class names that are conflicting classNames in the `openmrs-owa-orderentry.css` and there is also the argument for the concept of code/functional modularity.
However, if the team agrees we implement it this way, then I will gladly accept @Betty-Kebenei @segunolalive @fuchodeveloper @zeze111 @geofrocker @flacode

Reply:
Okay, so the team has agreed that we have all the css styles in one place and a Jira ticket for this purpose has been created because this change has to be implemented in a couple of places/files. Please look forward to that implementation in the future. Thanks